data/assets/examples/renderable/renderabledistancelabel/renderabledistancelabel_example.asset

```diff
@@ -0,0 +1,47 @@
+local moon = asset.require("scene/solarsystem/planets/earth/moon/moon")
```

Comment:
We decided to have the base name as the asset name.  So in this case it would be `/renderable/renderabledistancelabel/distancelabel.asset` as the file name

Reply:
We should have a description at the top of the file describing what the example does